### PR TITLE
Changelog for master

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @nickanderson can review this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,7 +26,6 @@ Notable changes to the framework should be documented here
 	- fix use of _stdlib_path_exists_<command> in FR transport_user policy
 	  bundle (ENT-4906)
 	- partitioned __inventory table for federated reporting (ENT-4842)
-	- psql_wrapper needed full path to psql binary (ENT-4912)
```
Comment (nickanderson) [inline CHANGELOG.md]: Actually FR was released in 3.14, so you should not take this edit, it should remain in the changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
+	- Fixed Python 3 incompatibility in yum package module
+	- Fixed synchronization of important configuration files from active to
+	  passive hub (ENT-4944)
+	- Made keys of all types from feeder hubs trusted on a superhub (ENT-4917)
+	- Speeded-up FR import process by merging INSERT INTO statements (ENT-4839)
+	- Suppressed stderr output from lldpctl when using path defined by
+	  def.lldpctl_json (CFE-3109)
+	- added SQL to update feeder update timestamp during import (ENT-4776)
+	- added ssh_home_t type to cftransport .ssh dir (ENT-4906)
+	- disabled federation policy test for now, broken in jenkins so needs
```
Comment (craigcomstock) [inline CHANGELOG.md]: This could probably be left out.

Comment (craigcomstock) [reply CHANGELOG.md]: also period. mea culpa. ;)

Comment (Lex-2008) [reply CHANGELOG.md]: right! Nick also deleted it in his commit

**Pull request merged**